i18n/en.json

```diff
@@ -153,7 +153,7 @@
   },
   {
     "id": "api.channel.add_member.added",
-    "translation": "%v added to the channel by %v"
+    "translation": "%v added to the team by %v"
```

Comment:
Hi @hmhealey. This resolve case 1 of #6483. Should I leave it as is or create a new string like what you just suggest for user deletion. I was not really sure so I added it anyway.

Reply:
Yeah, we should add the new string for the Town Square and change this one back to what it was before. Otherwise, it'll print "User added to the team" any time someone is added to any channel.